### ISSUE TITLE
Multiple hooks on same function not supported

### ISSUE THREAD

**Issue by stevemk14ebr** [stevemk14ebr/PolyHook#4]
Multiple detour hooks on the same function will crash, this may be fixed in a future update

**Comment by Karaulov** [comment]
Yes please fix! Need just read jmp addr and install hook at this addr! Please...:)

**Comment by stevemk14ebr** [comment]
Stop, i get it you want this. I've told you what my plans are, respect them or i will ban you.

**Comment by Karaulov** [comment]
"will ban you" 
Do you get pleasure from it? :D
